docs/prod-install.md

```diff
@@ -39,27 +39,33 @@ with the following commands:
 
 ```
 sudo -i  # If not already root
-cd /root
+cd $(mktemp -d)
 wget https://www.zulip.org/dist/releases/zulip-server-latest.tar.gz
-rm -rf /root/zulip && mkdir /root/zulip
-tar -xf zulip-server-latest.tar.gz --directory=/root/zulip --strip-components=1
+tar -xf zulip-server-latest.tar.gz
 ```
 
 Then, run the Zulip install script:
 ```
-/root/zulip/scripts/setup/install
+./zulip-server-*/scripts/setup/install
```

Comment:
Rereading the instructions, I think this separation into two sentences is actually helpful. It makes it a little clearer how the first bit can be any way of obtaining a release tarball and getting it onto the machine, and it helps make sense of the "just fix and rerun if it fails" instructions.

The `sudo` part can move down to the install, though, and I think that will make things slightly clearer. I'll do that.